mmcv/image/geometric.py

```diff
@@ -3,6 +3,7 @@
 
 import cv2
 import numpy as np
+from torch.nn.modules.utils import _pair as to_2tuple
```

Comment:
try not to add it here in case parrots do not support this internal API.

Reply:
How about we implement it in MMCV?

Reply:
> How about we implement it in MMCV?

agree

Reply:
Refer to this [PR](https://github.com/open-mmlab/mmcv/pull/1125)